packages/clerk-js/src/utils/captcha.ts

```diff
@@ -76,9 +74,12 @@ export async function loadCaptcha(url: string) {
   if (!window.turnstile) {
     try {
       await loadScript(url, { defer: true });
-    } catch (_) {
+    } catch {
       // Rethrow with specific message
-      clerkFailedToLoadThirdPartyScript('Cloudflare Turnstile');
```

Comment:
🙃 Why did we drop the usage of `clerkFailedToLoadThirdPartyScript` ? Couldn't we enhance it with an optional `url` parameter?

Reply:
As described above, the `clerkFailedToLoadThirdPartyScript` throws an Error object which is not what we want to do here.